addons/spreadsheet/static/src/global_filters/index.js

```diff
@@ -63,6 +63,15 @@ inverseCommandRegistry
                 filter: {},
             },
         ];
+    })
+    .add("MOVE_GLOBAL_FILTER", (cmd) => {
+        return [
+            {
+                type: "MOVE_GLOBAL_FILTER",
+                id: cmd.id,
+                delta : cmd.delta*-1
+            }
+        ]
```

Comment:
```suggestion
        return [
            {
                type: "MOVE_GLOBAL_FILTER",
                id: cmd.id,
                delta: cmd.delta *- 1,
            },
        ];
```
Don't forget to install the web tooling, otherwise the next one that goes through this file will reformat it